package.json

```diff
@@ -54,6 +54,7 @@
     "@synthetixio/synpress": "^3.5.1",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^14.0.0",
+    "@types/chrome": "^0.0.235",
```

Comment:
Why do we need it?

Reply:
It's needed to support this API: `chrome.scripting.registerContentScripts`